packages/frontend/core/src/components/blocksuite/block-suite-header/menu/index.tsx

```diff
@@ -81,11 +86,12 @@ export const PageHeaderMenuButton = ({
   const { favorite, toggleFavorite } = useFavorite(pageId);
 
   const { duplicate } = useBlockSuiteMetaHelper();
-  const { importFile } = usePageHelper(docCollection);
+  const { importFile, importSnapshotFile } = usePageHelper(docCollection);
```

Comment:
move the importSnapshotFile method to SnapshotMenuItems? In this way, we don’t have to pass props layer by layer, and we want to abandon usePageHelper and can’t add code to it anymore.